frontend/src/pages/docs/[...slug].tsx

```diff
@@ -312,11 +316,11 @@ const DocContent = styled.div`
   }
 
   & h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
+  & h2,
```

Comment:
@seve I got a stylelint error saying &h1 needs to go after h2, h3, h4, etc., but I assume that you wanted `&` for all header tags?